src/pages/SwapV2/index.tsx

```diff
@@ -529,7 +529,7 @@ export default function Swap({ history }: RouteComponentProps) {
     findTokenPairFromUrl()
     refIsImportUserToken.current = false
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [JSON.stringify(defaultTokens), refIsCheckNetworkAutoSelect.current])
+  }, [defaultTokens, refIsCheckNetworkAutoSelect.current])
```

Comment:
Why do you update it?     I saw that stringify is better in this case.   Actually, the swap page is currently 'slow' , I try to prevent re-render as few as possible.       You can test it by console.log defaultTokens in that useEffect, it logs more than without stringify.    Because  defaultTokens is  different object with same property {a:...} !== {a:...} => that is the reason why I use stringify .

Reply:
You are correct, `JSON.stringify` helps fewer re-render. But it is critical slow down when the object is large and very large (1000 tokens and 6000 tokens)

Reply:
@nguyenhoaidanh yes, this PR is not done yet, it still have space to improve. You can help contribute to it.

Reply:
nice catch @namgold 
anw, I dont recommend json.stringify, if you need push an object to dependencies, that object should memorize with useMemo

Reply:
true, if any object being memorized somewhere, no need json.stringify.

Reply:
So our first action: find source of this object and memorize them from the source

Reply:
Do you guys have any idea on debouncing update for re-render?